packages/flutter_driver/lib/src/common/find.dart

```diff
@@ -24,7 +24,7 @@ mixin DeserializeFinderFactory {
       case 'Descendant': return Descendant.deserialize(json, this);
       case 'Ancestor': return Ancestor.deserialize(json, this);
     }
-    throw DriverError('Unsupported search specification type $finderType');
+    return null;
```

Comment:
Why this change? This turns an error with a clear message into a type error if the `!` fails

Reply:
Good point, it was unnecessarily nullable. Reverted.